bundle/language_en.properties

```diff
@@ -726,10 +730,15 @@ angal.inventory.validated
 angal.inventory.done                                                                                   = Done
 angal.inventory.validate.btn                                                                           = Validate
 angal.inventory.validate.btn.key                                                                       = V
-angal.inventory.validatednoteditable.msg                                                               = A validated inventory cannot be modified.
+angal.inventory.inventorymustsavebeforevalidation.msg                                                    = Inventory must be saved before validation.
+angal.inventory.validate.success.msg                                                                   = The inventory has been successfully validated.
+angal.inventory.theoreticalqtyhavebeenupdatedforsomemedical.msg                                        = Theoretical quantities have been updated for certain drugs.
+angal.inventory.validate.error.msg                                                                     = Error during inventory validation.
+angal.inventory.allinventoryrowshouldhavelotbeforevalidation.msg                                         = All the inventory rows should have lots before validation.
 angal.inventory.viewinventory.title                                                                    = View Inventory
 angal.inventory.youhavealreadyaddedallproduct.msg                                                      = You have already added all the products.
 angal.inventoryrow.realqty.col                                                                         = Real Qty
+angal.inventory.sometheoriticqtyhavebeenupdatepleasecontrole.msg                                       = Some theoretical(s) quantity(ies) have been updated, please control all inventory(ies) row(s) before confirm this inventory.
```

Comment:
I would rather state:

`angal.inventory.sometheoreticqtyhavebeenupdatepleasecheckthefollowing.msg = Some theoretical quantities have been updated, please check the following inventory row(s) and validate again.`